release-content/release-notes/bevy_text_input_module.md

```diff
@@ -0,0 +1,34 @@
+---
+title: `bevy_text::input` module
+authors: ["@Ickshonpe"]
+pull_requests: [20366]
+---
+
+A long-standing feature request from our users is support for text input. Whether the user is creating a new character, logging in with a username and password, or creating a new save file, it's vitally important for them to be able to enter a string of text. Unfortunately, writing a robust and feature-rich text input widget is not easy, especially one that supports all of the expected capabilities (such as undo, range selection, and scrolling). This effort is made much easier now that Bevy has incorporated the cosmic crate for text handling.
+
+Features:
```

Comment:
Let's add:

"Support for bidirectional text, including RTL scripts such as Arabic and Hebrew."